src/main/java/capybara/Capybara.java

```
@@ -0,0 +1,67 @@
+package capybara;
+
+import capybara.command.Command;
+
+/**
+ * Entry point of the Capybara application.
+ * Coordinates initialization of UI, storage, and task list, and
+ * runs the main command loop until the user exits.
+ */
+public class Capybara {
+    private final Storage storage;
+    private final TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Constructs a new Capybara instance with the given storage file path.
+     *
+     * @param filePath Path to the storage file where tasks are saved.
+     */
+    public Capybara(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+
+        TaskList loaded;
+        try {
+            loaded = new TaskList(storage.load());
+        } catch (Exception e) {
```

Similar to my earlier comment, it is generally good practice to be specific when using exceptions. For example, in this case this try/catch block may catch more than you initially designed for and is a typical trait of exception swallowing.